src/services/db.ts

```diff
@@ -198,7 +198,8 @@ class DBCollection<TSchema extends Document = Document> {
 
   async cfg_indexes(): Promise<void> {
     this.log('Configure indexes.');
-    const indexes_config = _.get(cfg.db.collections[this.cfg_entry], 'indexes');
+    const collection_config = cfg.db.collections[this.cfg_entry];
+    const indexes_config = 'indexes' in collection_config ? collection_config.indexes : undefined;
```

Comment:
Shouldn't this be enough?
```suggestion
    const indexes_config = collection_config['indexes'];
```
That should make `indexes_config === undefined` if `'indexes'` is not present.

Reply:
Yes, you are right. Interesting, that it works right now. Before eslint wasn't happy.